apps/music/test/unit/player_view_test.js

```diff
@@ -0,0 +1,105 @@
+/* global PlayerView */
+
+'use strict';
+
+require('/js/ui/views/player_view.js');
+require('/shared/js/l10n.js');
+
+suite('Star Rating Bar Accessibility Test', function () {
+
+    var pv,
+    ratings;
+
+    suiteSetup(function () {
+       
```

Comment:
Ahh, I see what you mean now.  I've installed a plugin into my editor (Netbeans) to help me see that in the future.  Made sure to remove those.